implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -203,7 +203,21 @@ impl CreateCommand {
 
         opts.terminal
             .stdout()
-            .plain(
+            .plain(if let Some(last_failure) = inlet.last_failure.as_deref() {
+                fmt_err!(
+                    "TCP Inlet {} on node {} failed to connect to the outlet at {}\n",
+                    &cmd.from
+                        .to_string()
+                        .color(OckamColor::PrimaryResource.color()),
+                    &node.node_name().color(OckamColor::PrimaryResource.color()),
+                    &cmd.to
+                        .to_string()
+                        .color(OckamColor::PrimaryResource.color())
+                ) + &fmt_log!(
+                    "Error: {}\n",
+                    last_failure.color(OckamColor::Failure.color())
```

Comment:
This message can be misleading and confusing. We could log it (you already [added](https://github.com/build-trust/ockam/pull/7587/files#diff-5b3f34a98f0adf81c262908c1b6ee444f6e61e4242075b18860c9a57ea986e3fR409) this right?) but not write it to stdout.

Reply:
Maybe i can show the "last failure" when listing/showing the inlets

Reply:
After some experimentation, I'm completely removing `last_failure`. The errors are too rough to for the stdout.